components/server/src/workspace/gitpod-server-impl.ts

```diff
@@ -2345,7 +2345,12 @@ export class GitpodServerImpl implements GitpodServerWithTracing, Disposable {
         }
         await this.guardProjectOperation(user, projectId, "get");
         try {
-            return await this.projectsService.getProjectOverviewCached(user, project);
+            const result = await this.projectsService.getProjectOverviewCached(user, project);
+            if (result) {
+                const isProjectConsideredInactive = await this.projectsService.isProjectConsideredInactive(project.id);
+                result.prebuildState = isProjectConsideredInactive ? "suspended_on_inactivity" : "enabled";
```

Comment:
I would really prefer a simpler, more precise field for this (easier to understand, no "magic strings", fewer chances of misusing this abstraction later, etc)

```suggestion
                result.isProjectConsideredInactive = isProjectConsideredInactive;
```